src/renderer/atlas/BuiltinGlyphs.cpp

```diff
@@ -1183,13 +1177,13 @@ void BuiltinGlyphs::DrawBuiltinGlyph(ID2D1Factory* factory, ID2D1DeviceContext*
         }
         case Shape_FilledEllipsis:
         {
-            const D2D1_ELLIPSE e{ { begXabs, begYabs }, endX, endY };
+            const D2D1_ELLIPSE e{ { rectX + begX, rectY + begY }, endX, endY };
```

Comment:
why this change? it doesn't take into account the line offsets any longer

Reply:
Filled shapes don't have any line width, so this will work just fine.
This change ensures that the rounded powerline glyphs aren't slightly cut off anymore.

Reply:
okay but the empty glyphs DO have a line width - how does this apply on line 1186R?

Reply:
(looks fine on my screen)

Reply:
Ah right. More generally speaking, the `lineOffsetX/Y` code is only for snapping the coordinates of lines. In case of the "arc" shape glyph, this probably doesn't matter much because the entire glyph uses a lot of anti-aliasing anyway.